decidim-core/lib/decidim/core.rb

```diff
@@ -553,6 +553,17 @@ def self.reset_all_column_information
     "/"
   end
 
+  # This is the maximum time that the cache will be stored. If nil, the cache will be stored indefinitely.
+  # Currently, cache is applied in the Cells where the method `cache_hash` is defined.
+  config_accessor :cache_expiry_time do
```

Comment:
@andreslucena , I am discussing with @microstudi , and we are thinking to add a 24 hour cache TTL. This should expire the cache, skipping the issues with image loading and others ... what are your thoughts on this ?

Reply:
We set the default to 24 hours, and if needed we adjust it at a later date.